numpy/core/numeric.py

```diff
@@ -1619,28 +1619,36 @@ def indices(dimensions, dtype=int):
         The shape of the grid.
     dtype : dtype, optional
         Data type of the result.
+    sparse : boolean, optional
+        Return a sparse representation of the grid instead of a dense
+        representation. Default is False.
 
```

Comment:
Oh, @Ablinne I forgot, this needs a `.. versionadded:: 1.17` (one empty line, before and after). I suppose depending on how fast 1.17 is, this may also be in 1.18, since I guess we should feature freeze soon (although we rarely did a hard freeze).

Reply:
Like this?